pkgs/os-specific/darwin/yabai/default.nix

```diff
@@ -26,7 +26,7 @@ stdenv.mkDerivation rec {
     '';
     homepage = "https://github.com/koekeishiya/yabai";
```

Comment:
could you add the description as well

Reply:
There's already description

Reply:
damn sry.

Reply:
All good 👍 Easy to miss when GitHub collapses non-differing portions of the file :)